server/routes/country.go

```diff
@@ -91,3 +91,25 @@ func WinRatio(c *gin.Context) {
 	c.JSON(200, countries)
 
 }
+
+func WinRatioWithCountry(c *gin.Context) {
+    country := c.Param("country")
+	dot := initalize.GetInstance()
+	rows, errAllCountries := dot.Query(initalize.GetDB(), "Country-Win-Ratio-specific-country", country)
+	if errAllCountries != nil {
+		panic(errAllCountries.Error())
+	}
+	defer rows.Close()
+	countries := []types.WinRatio{}
```

Comment:
```suggestion
	countries := types.WinRatio{}
```

winRatio with Country would only return 1 value so you can get rid of the array and do something similar to other endpoints that only returns one thing